pymess/backend/__init__.py

```diff
@@ -192,25 +191,6 @@ def send(self, recipient, content, related_objects=None, tag=None, template=None
             self.publish_message(message)
         return message
 
-    @transaction.atomic
-    def bulk_send(self, recipients, content, related_objects=None, tag=None, template=None, **kwargs):
```

Comment:
this maybe can work bud inside recipients can be numbers/e-mails for this specific backend

Reply:
it's inside controller now